### PR TITLE
Dev 3351 feat/approval docs

### PR DESCRIPTION
**Pull request by Achalhii**
In this feat, we add a new UI:Schema (`ui:approvalDoc`) in this repo, and we add the possibility to define class names in the docs spec

+ new class : bg-blue

Wait merge of this PR to see the good preview

### CODE REVIEW

File: src/components/AutomatorService/AutomatorService.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ class AutomatorService extends Component {
           return (
             <div>
               <Tag>{job.status}</Tag>
-              {job.params?.origin === 'caas-styleguide' && (
+              {job.params?.origin === "caas-styleguide" && (
```
Comment (Achalhii) [inline src/components/AutomatorService/AutomatorService.js:78]:
```suggestion
              {job.params?.origin === 'caas-styleguide' && (
```

File: src/components/DocsService/ResourceForm.js [hunk 2]
```diff
@@ -311,4 +282,70 @@ ResourceForm.defaultProps = {
   customWidgets: {},
 };
 
+function getActions(service, resourceName) {
```
Comment (Fluorz) [inline src/components/DocsService/ResourceForm.js:285]: tu peux utiliser des ternaires à la place de faire des arrays à render

**Review by Fluorz — CHANGES_REQUESTED** [review]

**Comment by gabila-r** [comment]
What is this git history ? 😨

**Comment by Achalhii** [comment]
> What is this git history ? 😨

FF...

File: src/components/DocsService/ResourceForm.js (hunk 2 above)
Comment (Achalhii) [reply src/components/DocsService/ResourceForm.js:285]: L'array n'est pas envoyé à un render mais au params d'un component react https://ant.design/components/card#:~:text=of%20the%20Card-,Array%3CReactNode%3E,-%2D

**Review by Fluorz — APPROVED** [review]

File: src/components/DocsService/ResourceForm.js (hunk 2 above)
Comment (Fluorz) [reply src/components/DocsService/ResourceForm.js:285]: https://stackoverflow.com/a/47771259

**Pull request merged**